frontend/client/types.ts

```diff
@@ -57,6 +65,7 @@ export type ServiceTemplate = {
   service_version: string;
   home_chain_id: string;
   configurations: { [key: string]: ConfigurationTemplate };
+  service_env_variables: ServiceEnvVariable[];
```

Comment:
I am unsure if it's better a dictionary rather than an array? This will make somewhat easy find duplicates.

Reply:
Fair point. Then I'll make them as dictionaries. One question: should I use the `name` for keys, and remove it from the values? (If they are always the same, then we can avoid this duplication.)

Reply:
No, keys can be anything. For convention, I would use the lowercase service_env_variable name (but it can be anything, and it shoudl be ignored when processing).

Reply:
Don't remove anything from the values!